components/formFields.styles.js

```diff
@@ -48,7 +48,7 @@ export const styles = StyleSheet.create({
     marginLeft: 0,
     borderRadius: BORDERRADIUS.lg,
   },
-  customCheckBoxContainerParent: {
+  customCheckBoxContainer: {
```

Comment:
I'm not sure on the naming here as the above styles are technically the container styles, but you are right Parent is a bit confusing, maybe `customCheckBoxesContainer`? god I hate naming things :(

Reply:
maybe now that it is containing the specific Label component, i might change it to something specific in its own file.